components/definitions/rewardsInternals.d.ts

```diff
@@ -0,0 +1,20 @@
+declare namespace RewardsInternals {
+  export interface ApplicationState {
+    rewardsInternalsData: State | undefined
+  }
+
+  export interface State {
+    info: {
```

Comment:
nit: maybe just have one level? (so that we don't need info wrapper)

Reply:
Makes sense